code/modules/atmospherics/machinery/components/fusion/hfr_procs.dm

```diff
@@ -446,6 +509,29 @@
 
 	qdel(src)
 
+/**
+ * Induce hallucinations in nearby humans.
+ *
+ * force will make hallucinations ignore meson protection.
+ */
+/obj/machinery/atmospherics/components/unary/hypertorus/core/proc/induce_hallucination(strength, delta_time, force=FALSE)
+	for(var/mob/living/carbon/human/human in view(src, HALLUCINATION_HFR(heat_output)))
+		if(force || !istype(human.glasses, /obj/item/clothing/glasses/meson))
+			var/distance_root = sqrt(1 / max(1, get_dist(human, src)))
+			human.hallucination += strength * distance_root * delta_time
+			human.hallucination = clamp(human.hallucination, 0, 200)
+
+/**
+ * Emit radiation
+ */
+/obj/machinery/atmospherics/components/unary/hypertorus/core/proc/emit_rads(radiation)
+	rad_power = clamp((radiation / 1e5), 0, FUSION_RAD_MAX)
```

Comment:
why the ((), , ) here?

Reply:
cut paste from old code, will fix
https://github.com/tgstation/tgstation/pull/61593/commits/182031a3a34951e9daf6606463fc6566e8abb20c#diff-37c3808ea6375f06938c30e3eb4afc68f3f4157db9676aed9fa06c2b197d6febL672